docs/configuration/backends/kubernetes.md

```diff
@@ -81,6 +81,20 @@ See also [Kubernetes user guide](/user-guide/kubernetes).
 # Default: <built-in template>
 #
 # filename = "kubernetes.tmpl"
+
+# Enable IngressEndpoint configuration.
+# This will allow Traefik to update the status section of ingress objects, if desired.
+#
+# Optional
+#
+# [kubernetes.ingressEndpoint]
+#
+# One must be configured.
+# Publishedservice will override the hostname and ip settings if configured.
+#
+# hostname = "localhost"
+# ip = "127.0.0.1"
+# publishedService = "default/cheddar"
```

Comment:
Can we document that it's namespace/servicename? Or do you think this is clear to Kubernetes users? I didn't get it in the first moments. Maybe simply a different example would help like `publishedService = namespaceName/serviceName`